html-forms/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Html Forms</title>
+</head>
+<body>
+  <form>
+    <div>Email</div>
+    <input type="email" name="email"><br/>
+    <div>Why is coding cool?</div>
```

Comment:
This should be a `label` element, with a `for` attribute corresponding to an `id` on the target input element.